playbooks/adhoc/preflight/check.yml

```diff
@@ -0,0 +1,32 @@
+---
+- hosts: OSEv3
+  gather_facts: no
+  roles:
+    - openshift_preflight/facts
```

Comment:
I followed [this idea](https://groups.google.com/d/topic/ansible-project/2as7VmaoHXY/discussion) from the Ansible mailing-list to avoid growing arbitrarily the number of top level roles, while relying on just a common prefix to group them together.

In particular, I don't see these "roles" being usable by themselves, so it seemed more like a sub-role idea than multiple _reusable roles_ (what I understand to be the idea behind Ansible Roles).

TBH I'm not even super comfortable that the current format can scale well and be maintainable. So I'd think to be wise to allow for some imperfection, and embrace refactors instead of trying to get everything right on the first shot. We do need to write more checks and understand better the requirements and constraints of what we're doing to be able to steer the implementation.